### ISSUE TITLE
Install failed to include bin/podispell

### ISSUE THREAD

**Issue by mlawren** [haarg/podispell#1]
I just installed App::podispell on Debian, which failed to include the podispell script. Build log:

```
cpanm (App::cpanminus) 1.7039 on perl 5.020002 built for i586-linux-gnu-thread-multi-64int
Work directory is /home/mark/.cpanm/work/1438855472.17891
You have make /usr/bin/make
You have LWP 6.13
You have /bin/tar: tar (GNU tar) 1.27.1
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>.
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by John Gilmore and Jay Fenlason.
You have /usr/bin/unzip
Searching App::podispell () on cpanmetadb ...
--> Working on App::podispell
Fetching http://www.cpan.org/authors/id/H/HA/HAARG/App-podispell-0.001001.tar.gz
-> OK
Unpacking App-podispell-0.001001.tar.gz
Entering App-podispell-0.001001
Checking configure dependencies from META.json
Checking if you have ExtUtils::MakeMaker 6.58 ... Yes (6.98)
Configuring App-podispell-0.001001
Running Makefile.PL
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for App::podispell
Writing MYMETA.yml and MYMETA.json
-> OK
Checking dependencies from MYMETA.json ...
Checking if you have Test::More 0 ... Yes (1.001003)
Building and testing App-podispell-0.001001
cp lib/App/podispell.pm blib/lib/App/podispell.pm
Manifying blib/man3/App::podispell.3pm
PERL_DL_NONLAZY=1 /usr/bin/perl "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/00-load.t .. ok
All tests successful.
Files=1, Tests=1,  0 wallclock secs ( 0.04 usr  0.00 sys +  0.03 cusr  0.00 csys =  0.07 CPU)
Result: PASS
Installing /home/mark/.direnv/perl5/lib/perl5/App/podispell.pm
Installing /home/mark/.direnv/perl5/man/man3/App::podispell.3pm
Appending installation info to /home/mark/.direnv/perl5/lib/perl5/i586-linux-gnu-thread-multi-64int/perllocal.pod
-> OK
Successfully installed App-podispell-0.001001
Installing /home/mark/.direnv/perl5/lib/perl5/i586-linux-gnu-thread-multi-64int/.meta/App-podispell-0.001001/install.json
Installing /home/mark/.direnv/perl5/lib/perl5/i586-linux-gnu-thread-multi-64int/.meta/App-podispell-0.001001/MYMETA.json
1 distribution installed
```

I don't know how you generate Makefile.PL, but it doesn't seem to include a reference to bin/podispell, EXE, ...

**Comment by haarg** [comment]
That's a rather embarrassing bug.  Fixed in 9b8958e492c6d77ce15441a28ad5fcbd16e7f30b and released in 0.001002.

**Comment by mlawren** [comment]
> That's a rather embarrassing bug.

Not to me. I've done similar things quite often and fixed them, and
then fixed the fix, and then repeated everything again :-)

Thanks for the quick response.

## 

Mark Lawrence
